prow/jobs/incubator/compass/tests/compass-integration/compass-integration.yaml

```diff
@@ -0,0 +1,67 @@
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+job_template: &job_template
+  skip_report: false
+  decorate: true
+  path_alias: github.com/kyma-incubator/compass
+  max_concurrency: 10
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/test-infra/buildpack-golang:go1.11
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-incubator/compass/tests/compass-integration"
```

Comment:
Do we need to name the directory `compass-integration`? Maybe just `integration` is fine, or even `end-to-end`, similarly to `kyma`. It depends on which tests we want to have. Let's discuss it within our team.